### PR TITLE
Update integration-tests.md

### PR DESCRIPTION
**Pull request by vuqtran88**
Add visual studio requirement message.

### CODE REVIEW

File: docs/integration-tests.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Tests the integration of the New Relic .NET agent with various .NET applications. This test suite will only run on Windows.
 
 ## Installation
+Require Visual Studio 2019 Version 16.8 or greater.
```
Comment (angelatan2) [inline docs/integration-tests.md:6]: Requires instead of require.

**Review by angelatan2 — CHANGES_REQUESTED** [review]

File: docs/integration-tests.md (hunk 1 above)
Comment (vuqtran88) [reply docs/integration-tests.md:6]: 👍

Comment (vuqtran88) [reply docs/integration-tests.md]: fixed

**Review by nr-ahemsath — APPROVED** [review]

**Review by nr-ahemsath — APPROVED** [review]
👍

**Pull request merged**